app/modules/import-export-customization/processes/import.php

```diff
@@ -213,6 +213,7 @@ private function set_import_object( array $instance ) {
 		$this->manifest = $instance_data['manifest'];
 		$this->site_settings = $instance_data['site_settings'];
 
+		$this->kit_id = $instance_data['kit_id'] ?? '';
```

Comment:
I have updated the kit references, because the kit and thumbnail weren't passed correctly. This thumbnail reference was missing inside `import-export-customisation`.


<img width="694" height="561" alt="Screenshot 2025-10-22 at 11 02 55" src="https://github.com/user-attachments/assets/f270bab9-d92d-4f28-adcd-cd43248a6c86" />